accesspoc/search/management/commands/index_data.py

```diff
@@ -0,0 +1,34 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from elasticsearch.helpers import streaming_bulk
+from elasticsearch_dsl import Index
+from elasticsearch_dsl.connections import connections
+from tqdm import tqdm
+
+from dips.models import Collection, DIP, DigitalFile
+
+# Models to index in ES.
+MODELS = [Collection, DIP, DigitalFile]
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **kwargs):
+        es = connections.get_connection()
+        for model in MODELS:
+            document = model.es_doc
+            index_name = document._doc_type.index
+            print('Indexing %s:' % index_name)
+            index = Index(index_name)
+            index.settings(**settings.ES_INDEXES_SETTINGS)
+            index.doc_type(document)
+            index.delete(ignore=404)
+            index.create()
+            progress_bar = tqdm(total=model.objects.count())
+            for _ in streaming_bulk(
+                es,
+                (obj.get_es_data() for obj in model.objects.all().iterator()),
+                index=index_name,
+                doc_type='doc',
+            ):
+                progress_bar.update(1)
+            progress_bar.close()
```

Comment:
When creating the indices in a fresh installation I see the following:

```
Indexing accesspoc_collections:
0it [00:00, ?it/s]
Indexing accesspoc_dips:
0it [00:00, ?it/s]
Indexing accesspoc_digital_files:
0it [00:00, ?it/s]
```

The output is a bit confusing. Can you make it clearer?

Reply:
Yeah, good call.